src/Controller/Base.php

```diff
@@ -465,4 +465,16 @@ protected function resources()
     {
         return $this->app['resources'];
     }
+
+    /**
+     * Add a middleware that sets the response caching parameters.
+     *
+     * @param int $maxAge Number of seconds to set `Cache-Control s-maxage`
+     */
+    protected function setResponseCaching($maxAge)
+    {
+        $this->app->after(function (Request $request, Response $response) use ($maxAge) {
+            $response->setSharedMaxAge($maxAge)->setPublic();
+        });
+    }
```

Comment:
This isn't good because it applies the middleware to all requests. It would be better to set on route definition so it only gets applied to that route. 

I'm also worried that boiling it down to `$maxAge` doesn't get enough flexibility. Maybe changing the method name to setResponseMaxAge instead that way another cache helper could be created if needed.